buildSrc/src/main/java/quilt/internal/tasks/lint/FieldNamingChecker.java

```diff
@@ -24,9 +24,13 @@ public void check(FieldEntry entry, EntryMapping mapping, Function<Entry<?>, Acc
             if (startsWithUppercase(mapping.targetName())) {
                 errorReporter.warning("atomic field starts with uppercase character '" + mapping.targetName().charAt(0) + "'");
             }
-        } else if (access.isStatic() && access.isFinal()) {
+        } else if (access != null && access.isStatic() && access.isFinal()) {
```

Comment:
I'd add an error when access is null, I think it only can happen when there is an invalid mapping